packages/markdown/processor/compile/table-head.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function TableHeadCompiler() {
+  const { Compiler } = this;
+  const { visitors } = Compiler.prototype;
+  visitors.tableHead = function compile(node) {
+    return visitors.tableCell.call(this, node);
+  };
+};
```

Comment:
(@gratcliff I wonder if this missing compiler had anything to do with issues re: [indexing table content in Algolia](https://app.asana.com/0/0/1181086456895394/f)?)

Reply:
I think our other table compiler should take care of it. From tabbing through some of their changelogs - they have some super long tables. We're capping individual index records at `2kb`, so they're most likely hitting that cap.

Reply:
Following up - it does seem like there's an actual bug. Let me take a look.

Reply:
@rafegoldberg 
[This](https://github.com/readmeio/api-explorer/blob/master/packages/markdown/processor/plugin/table-flattening.js)
seems to be returning joined empty values for that ticket :/

Reply:
Anyway! That's separate work. This looks good.